google/cloud/spanner_dbapi/connection.py

```diff
@@ -196,6 +203,24 @@ def read_only(self, value):
             )
         self._read_only = value
 
+    @property
+    def batch_mode(self):
+        """_batch_mode flag for this connection.
+
+        :rtype: bool
+        :returns: _batch_mode flag value.
+        """
+        return self._batch_mode
+
+    @batch_mode.setter
+    def batch_mode(self, value):
+        """`batch_mode` flag setter.
+
+        Args:
+            value (BatchMode)
+        """
+        self._batch_mode = value
```

Comment:
What happens if an external uses calls this function when the connection is already in the middle of a different type of batch (e.g. it is now in a DML batch, and it is called to set it to DDL batch)?

Reply:
Removed this property